docs/data.md

```diff
@@ -231,6 +275,12 @@ Attributes|Type|Use|Description
 `entityKey`|string|optional|Key attribute of the referenced entity from which to get a value.
 `unique`|--|--|*This attribute hasn't been implemented yet.*
 
+Example:
+
+```
```

Comment:
Use xml as the codeblock type.

Reply:
@dobooth This has been done. Please review it.